modules/sharedaddy/sharing-service.php

```diff
@@ -504,11 +504,16 @@ function sharing_add_footer() {
 		if ( apply_filters( 'jetpack_sharing_counts', true ) && is_array( $jetpack_sharing_counts ) && count( $jetpack_sharing_counts ) ) :
 			$sharing_post_urls = array_filter( $jetpack_sharing_counts );
 			if ( $sharing_post_urls ) :
+				$is_jetpack = true === apply_filters( 'is_jetpack_site', false, get_current_blog_id() );
```

Comment:
@aduth Would you mind adding a docblock before the filter? We're trying to document each filter and action in Jetpack, and it would be nice if we all started doing it with new filters while [some of us work on the existing ones](https://github.com/Automattic/jetpack/issues?utf8=%E2%9C%93&q=label%3Adocs+).

Here is an example: #1801

Thanks!